index.js

```diff
@@ -95,6 +101,8 @@ function magnetURIDecode (uri) {
   result.announce = Array.from(new Set(result.announce))
   result.urlList = Array.from(new Set(result.urlList))
 
+  if (result.x && result.x.pe) result.peerAddress = result.x.pe
```

Comment:
I would rather force `x.pe` to be a unique array (aka set) like `announce` and `urlList`.

Also I would change the name to `peerAddresses` (to reflect that it contains multiple addresses)

Reply:
I've added the code to remove duplicates 👍
Regarding `peerAddress`, given that it can contain a string to represent a single value or an array for multiple ones, I've decided to leave as it is described in BEP09 😅